### DISCUSSION TITLE
Can we have user preference in dashboard like we have it in theme.

### DISCUSSION THREAD

**Discussion by a-imran99** [grafana/grafana#44690] [Dashboards, Panels & Visualizations]
**Can we have a user preference at Dashboard level which is similar to user preference in theme.** 

When we change theme, either light or dark it is saved in user preference and  when particular user logs in the particular theme selected by the user is reflected. Similarly can we have same functionality in dashboards. when a particular user( who has editor role) edits a dashboard (rearranges or resize panel) that dashboard should be rendered to that particular user alone and not for other users. 

As of now, the changes made to dashboard by any user who has admin or editor role is reflected to one single dashboard which is rendered for all users. 

Is this functionality possible? please provide suggestion if there are other possibilities to achieve this!

**Comment by a-imran99** [comment]
@torkelo Any help on this?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
